lib/python/src/aria/ops/definition/credential_type.py

```diff
@@ -122,24 +122,28 @@ class CredentialEnumParameter(CredentialParameter):
     def __init__(
         self,
         key: str,
-        values: list[str],
+        values: list[Union[str, tuple[str, str]]],
         label: Optional[str] = None,
-        default: Optional[str] = None,
+        default: Optional[Union[str, tuple[str, str]]] = None,
```

Comment:
no reason to change this. If someone wants a label for the default, they should add it to 'values'

Reply:
What should generally happen is this:
```
enum_values: [("key1", "label1"), ("key2", "label2"), ("key3", "label3")]
default: "key2"
```
In which the default will have the label "label2", but we don't specify it in the 'default' section.

We'll also allow something like this
```
enum_values: [("key1", "label1"), ("key2", "label2"), ("key3", "label3")]
default: "key4"
```
In which the default will have the label "key4". It's possible we may want to not allow this case, but it would technically be a breaking change.

We want to not get into this situation:
``
enum_values: [("key1", "label1"), ("key2", "label2"), ("key3", "label3")]
default: ("key2", "default_label")
```
In the above, what is the label for 'key2' - "label2" or "default_label"? We can only set one.